src/test/librbd/test_mock_fixture.h

```diff
@@ -25,7 +25,11 @@ ACTION_P(CopyInBufferlist, str) {
 }
 
 ACTION_P2(CompleteContext, r, wq) {
+#if defined(__clang__)
+  ContextWQ *context_wq = static_cast<ContextWQ *>(wq);
```

Comment:
@dillaman 
My Linux VM is not really fast, so compiling will take some time.

I would expect so, but when I changed it in the template, GNUCC was not happy.
I'll get back here with GNUCC's complaints. (or not)